docs/pom.xml

```diff
@@ -25,6 +25,7 @@
 		<!--maven-resources-plugin:3.2.0:copy-resources fails: https://issues.apache.org/jira/browse/MSHARED-966-->
 		<maven-resources-plugin.version>3.1.0</maven-resources-plugin.version>
 		<!-- override version inherited from spring-cloud-build parent to one that is available on Maven Central -->
+		<spring-asciidoctor-backends.version>0.0.4</spring-asciidoctor-backends.version>
 		<spring-asciidoctor-extensions-block.version>0.6.1</spring-asciidoctor-extensions-block.version>
 		<spring-asciidoctor-extensions-boot.version>0.6.1</spring-asciidoctor-extensions-boot.version>
```

Comment:
From the change [here](https://github.com/spring-cloud/spring-cloud-build/pull/234/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8L36-L40), looks like these can also be removed?
```suggestion
```

Reply:
@zhumin8 Ah I wasn't able to remove these without the following error still:
```
[ERROR] Failed to execute goal org.asciidoctor:asciidoctor-maven-plugin:2.1.0:process-asciidoc (generate-html-documentation) on project spring-cloud-gcp-docs: Execution generate-html-documentation of goal org.asciidoctor:asciidoctor-maven-plugin:2.1.0:process-asciidoc failed: Plugin org.asciidoctor:asciidoctor-maven-plugin:2.1.0 or one of its dependencies could not be resolved: Failed to collect dependencies at org.asciidoctor:asciidoctor-maven-plugin:jar:2.1.0 -> io.spring.asciidoctor:spring-asciidoctor-extensions-spring-boot:jar:0.6.0
Failed to read artifact descriptor for io.spring.asciidoctor:spring-asciidoctor-extensions-spring-boot:jar:0.6.0: Could not transfer artifact io.spring.asciidoctor:spring-asciidoctor-extensions-spring-boot:pom:0.6.0 from/to spring-releases (https://repo.spring.io/release): Authentication failed for https://repo.spring.io/release/io/spring/asciidoctor/spring-asciidoctor-extensions-spring-boot/0.6.0/spring-asciidoctor-extensions-spring-boot-0.6.0.pom 401 Unauthorized
```
My understanding: these two plugin dependencies are still getting inherited from the parent pom's plugin config [here](https://github.com/spring-cloud/spring-cloud-build/blob/v3.1.7/pom.xml#L1142-L1150), just now unused, and [this change](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1914/files#diff-749f6f9897e14f3c05890960c13ad402b7541d714116175a61cada4e64f085a1R72-R76) in our child pom only adds `spring-asciidoctor-backends` as a plugin dependency instead of replacing the whole `<dependencies>` block (couldn't figure out a good way to achieve the latter).

Reply:
Ah. I see. Let's merge this in. Still it's great you were able to get rid of the local installation!